File: src/main/java/org/adler/P9PalindromeNumber.java
package org.adler;

class P9PalindromeNumber {
    public static void main(String[] args) {
        System.out.println("Problem 9 palindrome number");
        System.out.println(isPalindrome(121));
        System.out.println(isPalindrome(11));
    }

    public static boolean isPalindrome(int x) {
        if (x < 0) {
            return false;
        }

        String tmpStr = String.valueOf(x);
        String reversed = new StringBuilder(tmpStr).reverse().toString();

        return tmpStr.equals(reversed);
    }
}

File: TypeScript/P9PalindromeNumber.js
console.log("9. Palindrome Number");
console.log("Case 1: ", isPalindrome(121));
console.log("Case 2: ", isPalindrome(-121));
console.log("Case 3: ", isPalindrome(10));
var newArr = [1, "2"];
function isPalindrome(x) {
    if (x < 0) {
        return false;
    }
    var str = x.toString();
    var revertStr = str.split("").reverse().join("");
    if (str == revertStr)
        return true;
    return false;
}
